### PR TITLE
rtags 2.7

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by vladshablinsky** [comment]
I would say this is an upstream issue.
First of all, tests seem to be wrong and should be like this:

```diff
diff --git a/Formula/rtags.rb b/Formula/rtags.rb
index 9060175..ee12889 100644
--- a/Formula/rtags.rb
+++ b/Formula/rtags.rb
@@ -87,9 +87,9 @@ class Rtags < Formula
 
     begin
       sleep 1
-      pipe_output("#{bin}/rc -c", "clang -c src/foo.c", 0)
+      pipe_output("#{bin}/rc -c", "clang -c #{testpath}/src/foo.c", 0)
       sleep 1
-      assert_match "foo.c:1:6", shell_output("#{bin}/rc -f src/foo.c:5:3")
+      assert_match "foo.c:1:6", shell_output("#{bin}/rc -f #{testpath}/src/foo.c:5:3")
       system "#{bin}/rc", "-q"
     ensure
       Process.kill 9, rdm
```

But this doesn't change anything too much. Same commands from tests but executed manually work fine everywhere but under `/tmp` (which is `/private/tmp`) directory.
I tried different things. Tried to make `src/foo.c` right under my `/` 😬, tried creating new symlink to
`/private/tmp` and many other things, but never succeeded with anything linking to `/private/tmp`.

For now, without applying the diff I attached you can run tests successfully if you do this manually in a directory you run `brew test rtags` from.

```ruby
    mkpath testpath/"src"
    (testpath/"src/foo.c").write <<-EOS.undent
        void zaphod() {
        }

        void beeblebrox() {
          zaphod();
        }
    EOS
    (testpath/"src/README").write <<-EOS.undent
        42
    EOS
```

So, summing up, seems like an upstream issue.

**Comment by vladshablinsky** [comment]
On the other hand if we tested under other directory it would work fine, so I don't know how to call this issue.

**Comment by vladshablinsky** [comment]
https://github.com/Andersbakken/rtags/issues/860#issuecomment-263078346

**Comment by vladshablinsky** [comment]
@ilovezfs this should fix tests
```diff
diff --git a/Formula/rtags.rb b/Formula/rtags.rb
index 9c3e39d..c4bf030 100644
--- a/Formula/rtags.rb
+++ b/Formula/rtags.rb
@@ -82,14 +82,14 @@ class Rtags < Formula
     rdm = fork do
       $stdout.reopen("/dev/null")
       $stderr.reopen("/dev/null")
-      exec "#{bin}/rdm", "-L", "log"
+      exec "#{bin}/rdm", "--exclude-filter=\"\"", "-L", "log"
     end
 
     begin
       sleep 1
-      pipe_output("#{bin}/rc -c", "clang -c src/foo.c", 0)
+      pipe_output("#{bin}/rc -c", "clang -c #{testpath}/src/foo.c", 0)
       sleep 1
-      assert_match "foo.c:1:6", shell_output("#{bin}/rc -f src/foo.c:5:3")
+      assert_match "foo.c:1:6", shell_output("#{bin}/rc -f #{testpath}/src/foo.c:5:3")
       system "#{bin}/rc", "-q"
     ensure
       Process.kill 9, rdm
```
